samples/notebooks/openqasm.ipynb

```diff
@@ -0,0 +1,351 @@
+        "\n",
+        "### Run OpenQASM 3 Code in interactive session\n",
+        "\n",
+        "Import the `qsharp` module.\n",
+        "\n",
+        "This initializes a QDK interpreter singleton."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "75b8b81b",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "import qsharp\n",
+        "qsharp.init(target_profile=qsharp.TargetProfile.Base)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "5dc25416",
+      "metadata": {},
+      "source": [
+        "With the runtime initialized, we can import an OpenQASM program as a Python callable. Here we'll compile the OpenQASM program to a callable name `\"bell\"`."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "012cc902",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from qsharp.openqasm import import_openqasm\n",
+        "\n",
+        "source = \"\"\"\n",
+        "include \"stdgates.inc\";\n",
+        "bit[2] c;\n",
+        "qubit[2] q;\n",
+        "h q[0];\n",
+        "cx q[0], q[1];\n",
+        "c = measure q;\n",
+        "\"\"\"\n",
+        "\n",
+        "import_openqasm(source, name=\"bell\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "b95a0c8b",
+      "metadata": {},
+      "source": [
+        "We can now import it via the QDK's Python bindings and run it:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "db043bda",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from qsharp.code import bell\n",
+        "bell()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "8db074dc",
+      "metadata": {},
+      "source": [
+        "Additionally, since it is defined in the runtime, we can run it directly from a Q# cell:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "ad6d0331",
+      "metadata": {
+        "vscode": {
+          "languageId": "qsharp"
+        }
+      },
+      "outputs": [],
+      "source": [
+        "%%qsharp\n",
+        "bell()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "6578cadc",
+      "metadata": {},
+      "source": [
+        "This also unlocks all of the other `qsharp` package functionality. Like noisy simulation. Here we'll use Q#'s `run` function instead showing how we can call into the defined program from Q#:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "5bbd6d92",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from qsharp import run, DepolarizingNoise\n",
+        "from qsharp_widgets import Histogram\n",
+        "\n",
+        "Histogram(run(\"bell()\", shots=1000, noise=DepolarizingNoise(0.01)))"
```

Comment:
This works fine, but why not use the callable object like the other samples? I think there was a bug previously where this didn't work as expected, but now it seems to work just fine.
```suggestion
        "Histogram(run(bell, shots=1000, noise=DepolarizingNoise(0.01)))"
```

Reply:
The text above this line talks about how to call it just like you would in a %%qsharp cell. We can change this but we need to update more for consistency.